classes/help.php

```diff
@@ -207,19 +206,19 @@ function print_page() {
 
 												<tr>
 													<td>
-														<?php _e( 'To properly add a new location, you must enter one or both of the following:', 'SimpleMap' ); // XSS ok. ?>
+														<?php _e( 'To properly add a new location, you must enter one or both of the following:', 'simplemap' ); // XSS ok. ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 174 characters

<!-- Comment by [SideCI](https://sideci.com) -->